src/commands/NewCamelProjectCommand.ts

```diff
@@ -119,15 +140,28 @@ export abstract class NewCamelProjectCommand {
 	 * @param destPath Path of destination
 	 */
 	private async copyFile(sourcePath: string, destPath: string): Promise<void> {
-		if (workspace.workspaceFolders){
-			const wsPath = workspace.workspaceFolders[0].uri.fsPath;
-			const sourcePathUri = Uri.file(path.resolve(__dirname, sourcePath));
-			const destPathUri = Uri.file(path.join(wsPath, destPath));
-			try {
-				await workspace.fs.copy(sourcePathUri, destPathUri, { overwrite: false });
-			} catch (error) {
-				// Do nothing in case there already exists tasks.json and launch.json files
-			}
+
+		const sourcePathUri = Uri.file(path.resolve(__dirname, sourcePath));
+		const destPathUri = Uri.file(path.resolve(__dirname, destPath));
+		try {
+			await workspace.fs.copy(sourcePathUri, destPathUri, { overwrite: false });
+		} catch (error) {
+			// Do nothing in case there already exists tasks.json and launch.json files
 		}
+
+	}
+
+	/**
+	 * Shows a modal asking for user confirmation of a potential desctructive action in the selected folder.
+	 * VSCode automatically provides a 'Cancel' option which return `undefined`. The continue option will return the string `Continue`.
+	 *
+	 * @returns string | undefined
+	 */
+	private async confirmDestructiveActionInSelectedFolder() {
+		const message = 'Files in the selected folder WILL BE DELETED before project creation, continue?';
```

Comment:
it would be nice to include the path of the selected folder in the message (can be done in a separate PR as another iteration)